README.md

```diff
@@ -5,167 +5,142 @@
 [![Build Status][1]][2]
```

Comment:
On the sidebar, I don't see an explicit link to get back to README.md once you've navigated away (only clicking on the upper-left Heptio Ark gets there). Should/could we have a "Quickstart" link at the top of the nav to make this more obvious/easy

Reply:
I know :-( 
I've attempted several solutions to this problem, unsuccessfully so far. I'll keep playing.

Reply:
Attempt #N: wdyt?
![image](https://user-images.githubusercontent.com/6934230/36225019-5e839406-1197-11e8-9538-c10c05690468.png)